examples/kafka-authentication-sasl-plain-consumer/kafka_authentication_sasl_plain_consumer.out

```diff
@@ -1,5 +1,3 @@
-# To run this sample, navigate to the directory that contains the
-# `.bal` file, and execute the `bal run` command below.
 bal run kafka_authentication_sasl_plain_consumer.bal
 
```

Comment:
Shall we remove this extra empty line?
This might affect the BBE output verification. (Since messaging BBEs aren't verified, this will not be a big issue, but better to remove them).